tests/test_jobs/test_import_student_photos.py

```diff
@@ -47,7 +47,7 @@ def test_import_student_photos(self, app, metadata_db, student_tables, caplog):
             assert success_rows[0]['sid'] == '11667051'
 
             failure_rows = rds.fetch(f"SELECT * FROM {app.config['RDS_SCHEMA_METADATA']}.photo_import_status WHERE status = 'failure'")
-            assert len(failure_rows) == 0
+            assert len(failure_rows) == 1
```

Comment:
@lyttam - Looking back at the code for ImportStudentPhotos, it seems like it's reporting a failure (rather than simply returning no response) because it can't find an UID translation for this SID, which it does by checking the Calnet schema. Should the new user be mocked out in that schema `(in fixtures/students.sql)` as well?

Reply:
@paulkerschen yes, good call. I'll amend.